### ISSUE TITLE
[BE] 10-10EZR Form: RECEIVE/PRE-FILL Section I (Vet Info) EZR data

### ISSUE THREAD

**Issue by alexseelig** [department-of-veterans-affairs/va.gov-team#63861]
As a Veteran, I would like to be able to update as much of my information online as I can so that I can make changes easily without having to call the hotline and wait on hold or print out the form and snail mail it in.

- [ ] Build an API that can receive all EZR fields from the ES for **Section I** of the EZR (fields listed below)
- [ ] Make sure that that the information we receive can be used to pre-fill the EZR Form--we will have a separate story for the FE piece
- [ ] Test API to make sure all info is received from ES
- [ ] **Success: We have an API that can receive and pre-fill EZR Section I (Vet Info) info from ES into our app**

--------------------------------------------------
EZR Fields ([spreadsheet with all EZR fields](https://docs.google.com/spreadsheets/d/1IGLA8GT35ffS2PnFvCwdnA7bWTL5187TN9QlVwUXYr4/edit#gid=1410739912)):

**Section I - General Information**
Name (Read only for Veteran)
Preferred Name (will be behind feature flag on front end--VES cannot receive) 
Birth Sex (Read only for Veteran)
Gender Identity (will be behind feature flag on front end)
Social Security Number (Read only for Veteran)
Date of Birth (Read only for Veteran)
Mailing Address 
Home Telephone Number 
Mobile Telephone Number
E-mail Address 
Home Address 
Current Martial Status 

-----------------------------------------------
**Resources:**

- **[API Docs](https://ves.va.gov/esr-ws/spring-ws/getEESummary/eeSummary.wsdl)**
- [Lihan's Documentation](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/tutorials/record_vcr.md)
- [VA Profile API sharepoint](https://dvagov.sharepoint.com/sites/OITEPMOVAPROPUB/SitePages/Getting-Started.aspx)
- [Health Benefits Swagger Doc](https://qa.vaprofile.va.gov:7005/health-benefit/swagger-ui/index.html?configUrl=/health-benefit/v3/api-docs/swagger-config)
- [Personal Information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/personal_information/backend_documentation.md)
- [Contact Information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/contact_information/backend_documentation.md)
- [Military Information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/military_info/backend_documentation.md)
- [Direct Deposit Information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/direct_deposit/backend_documentation.md)
- [Notification Settings](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/notification_settings/backend_documentation.md)
- [Account Security](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/account_security/backend_documentation.md)

**Comment by alexseelig** [comment]
As a Veteran, I would like to be able to update as much of my information online as I can so that I can make changes easily without having to call the hotline and wait on hold or print out the form and snail mail it in.

- [ ] Build an API that can receive all EZR fields from the ES (EZR Sections I-VII) (fields listed below)
- [ ] Make sure that that the information we receive can be used to pre-fill the EZR Form
- [ ] Test API to make sure all info is received from ES
- [ ] **Success: We have an API prototype that can receive and pre-fill all EZR info from ES into our app**


--------------------------------------------------
EZR Fields ([spreadsheet with all EZR fields](https://docs.google.com/spreadsheets/d/1IGLA8GT35ffS2PnFvCwdnA7bWTL5187TN9QlVwUXYr4/edit#gid=0)):

Name (L, F, M)
Preferred (L, F, M)
SSN
Birth Sex
SIGI
Date of Birth
Home Phone
Mobile phone

Mailing address
City
State
Zip
County
Home address
City
State
Zip
County
Email address

Marital Status
"Health ins company 
(name, address, phone)"
Name of Policy Holder
Policy number
Group Code
Eligile for Medicaid?
Enrolled in Medicare Part A?
Effective date
Spouse name (L, F, M)
Spouse SSN
Spouse DOB
Spouse SIGI
Date of Marriage
"Spouse Address
City, state, Zip"
Child's name (L, F, M)
Child DOB
Child SSN
Date child became dependent
Child relationship to you
Child permanently and totally disabled before age 18
Child between 18 & 23 & attended school?
Expenses paid for child college, voc rehab or training
Did you provide support if they did not live with you?
Gross annual income from employment
Net income from farm, ranch, property or Biz
Other income
non-reimbursed medical expenses paid by you or spouse
amount paid last calendar year for funeral and burial
Amount paid last calendar year for your college or vocational education

-----------------------------------------------
**Resources:**

- **[API Docs](https://ves.va.gov/esr-ws/spring-ws/getEESummary/eeSummary.wsdl)**
- [Lihan's Documentation](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/tutorials/record_vcr.md)
- [VA Profile API sharepoint](https://dvagov.sharepoint.com/sites/OITEPMOVAPROPUB/SitePages/Getting-Started.aspx)
- [Health Benefits Swagger Doc](https://qa.vaprofile.va.gov:7005/health-benefit/swagger-ui/index.html?configUrl=/health-benefit/v3/api-docs/swagger-config)
- [Personal Information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/personal_information/backend_documentation.md)
- [Contact Information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/contact_information/backend_documentation.md)
- [Military Information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/military_info/backend_documentation.md)
- [Direct Deposit Information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/direct_deposit/backend_documentation.md)
- [Notification Settings](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/notification_settings/backend_documentation.md)
- [Account Security](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/account_security/backend_documentation.md)
